coretex/utils/docker.py

```diff
@@ -123,3 +124,17 @@ def imageInspect(image: str) -> Dict[str, Any]:
         raise TypeError(f"Invalid function result type \"{type(jsonOutput[0])}\". Expected: \"dict\"")
 
     return jsonOutput[0]
+
+
+def dockerInfo() -> Tuple[int, int, int]:
+    _, output, _ = command(["docker", "info"], ignoreStdout = True, ignoreStderr = True)
```

Comment:
`docker info` command can be formatted to json, please do so instead of using regex to manually parse values